docs/configuration/configuration.md

```diff
@@ -32,7 +32,7 @@ value is set to the specified default.
 Generic placeholders are defined as follows:
 
 * `<boolean>`: a boolean that can take the values `true` or `false`
-* `<duration>`: a duration matching the regular expression `[0-9]+(ms|[smhdwy])`
+* `<duration>`: a duration matching the regular expression `(([0-9]+)y)?(([0-9]+)w)?(([0-9]+)d)?(([0-9]+)h)?(([0-9]+)m)?(([0-9]+)s)?(([0-9]+)ms)?`
```

Comment:
This doesn't cover 0.

This is also beyond what a human can easily eyeball, so I'd include an example or two.

Reply:
In which case is 0 valid in the context of this file?

Reply:
it's a model.Duration, and they take 0 - whether it's a flag or a config field.

Reply:
done